### PR TITLE
Version 0.27.0

### PR DESCRIPTION
**Pull request by aminalaee**
### Added
* Minify JSON websocket data via `send_json` https://github.com/encode/starlette/pull/2128

### Fixed
* Correct the RuntimeError message content in websockets https://github.com/encode/starlette/pull/2141
* <TBA>

**Full Changelog**: https://github.com/encode/starlette/compare/0.26.1...0.27.0

# Checklist

- [ ] I understand that this PR may be closed in case there was no previous discussion. (This doesn't apply to typos!)
- [ ] I've added a test for each change that was introduced, and I tried as much as possible to make a single atomic change.
- [ ] I've updated the documentation accordingly.

### CODE REVIEW

**Comment by Kludex** [comment]
This is the right comparison: https://github.com/encode/starlette/compare/0.26.1...master

File: docs/release-notes.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+## 0.27.0
+
+May 14, 2023
+
+### Added
+* Minify JSON websocket data via `send_json` https://github.com/encode/starlette/pull/2128
+
+### Fixed
+* Correct the RuntimeError message content in websockets https://github.com/encode/starlette/pull/2141
+* TBA
```
Comment (Kludex) [inline docs/release-notes.md]:
```suggestion
```

File: docs/release-notes.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
+## 0.27.0
+
+May 14, 2023
+
+### Added
+* Minify JSON websocket data via `send_json` https://github.com/encode/starlette/pull/2128
+
+### Fixed
+* Correct the RuntimeError message content in websockets https://github.com/encode/starlette/pull/2141
```
Comment (Kludex) [inline docs/release-notes.md]:
```suggestion
* Replace `commonprefix` by `commonpath` on `StaticFiles` [1797de4](https://github.com/encode/starlette/commit/1797de464124b090f10cf570441e8292936d63e3).
* Convert ImportErrors into ModuleNotFoundError [#2135].(https://github.com/encode/starlette/pull/2135)
* Correct the RuntimeError message content in websockets [#2141](https://github.com/encode/starlette/pull/2141).
```

**Comment by aminalaee** [comment]
Wait for the new PR please

File: docs/release-notes.md [hunk 3]
```diff
@@ -1,3 +1,15 @@
+## 0.27.0
+
+May 14, 2023
```
Comment (Kludex) [inline docs/release-notes.md]:
```suggestion
May 16, 2023
```

**Comment by Kludex** [comment]
> Wait for the new PR please

What are you talking about?

**Comment by aminalaee** [comment]
I meant wait to add the new PR to release notes, it's looking good now 😄

File: docs/release-notes.md [hunk 4]
```diff
@@ -1,3 +1,15 @@
+## 0.27.0
+
+May 16, 2023
+
+### Added
+* Minify JSON websocket data via `send_json` https://github.com/encode/starlette/pull/2128
+
+### Fixed
+* Replace `commonprefix` by `commonpath` on `StaticFiles` [1797de4](https://github.com/encode/starlette/commit/1797de464124b090f10cf570441e8292936d63e3).
+* Convert ImportErrors into ModuleNotFoundError [#2135](https://github.com/encode/starlette/pull/2135)
```
Comment (Kludex) [inline docs/release-notes.md]:
```suggestion
* Convert ImportErrors into ModuleNotFoundError [#2135](https://github.com/encode/starlette/pull/2135).
```

**Comment by aminalaee** [comment]
https://github.com/encode/starlette/releases/edit/untagged-49fcb838b4706d597c3a

**Comment by Kludex** [comment]
Can you add a note in the release notes right below the date mentioning the advisory, and the link to it? Please

**Comment by Kludex** [comment]
It's super hard to do stuff on the phone

**Comment by aminalaee** [comment]
Sure, let me take care of this.

**Comment by Kludex** [comment]
Also, create a GitHub release draft, please.

**Comment by aminalaee** [comment]
I have created the draft and linked it here.

**Comment by Kludex** [comment]
Looks good

File: docs/release-notes.md [hunk 5]
```diff
@@ -1,3 +1,20 @@
+## 0.27.0
+
+May 16, 2023
+
+### Security Patch
```
Comment (Kludex) [inline docs/release-notes.md]:
```suggestion
```

File: docs/release-notes.md [hunk 6]
```diff
@@ -1,3 +1,19 @@
+## 0.27.0
+
+May 16, 2023
+
+
```
Comment (Kludex) [inline docs/release-notes.md]:
```suggestion
```

File: docs/release-notes.md [hunk 7]
```diff
@@ -1,3 +1,18 @@
+## 0.27.0
+
+May 16, 2023
+
+This release fixes a path traversal vulnerability in StaticFiles affecting versions >= 0.13.5. You can view the full security advisory:
```
Comment (Kludex) [inline docs/release-notes.md]:
```suggestion
This release fixes a path traversal vulnerability in `StaticFiles`. You can view the full security advisory:
```

**Review by Kludex — APPROVED** [review]

**Pull request merged**